.github/workflows/build_wheels_windows.yml

```diff
@@ -1,4 +1,4 @@
-name: Windows x86/x64
+name: Windows x86_x64
```

Comment:
I think it should be `x86_64` if putting together.

Reply:
Edited typing.